scripts/load_data.sh

```diff
@@ -0,0 +1,2 @@
+curl -L -o public/data/v2/getTermDump/allTerms.json searchneu.com/data/v2/getTermDump/allTerms.json
```

Comment:
As we talked about, the json's are likely going to be deprecated. We can probably pull direct from `elasticsearch.searchneu.com` endpoint, but this is totally good for now!